react/features/base/avatar/components/web/StatelessAvatar.js

```diff
@@ -66,6 +67,7 @@ export default class StatelessAvatar extends AbstractStatelessAvatar<Props> {
                     <img
                         alt = 'avatar'
                         className = { this._getAvatarClassName() }
+                        crossOrigin = { isGravatarURL(url) ? '' : undefined }
```

Comment:
Hum, on second thought ... can we detect it another way ... by default we do not use gravatar I think, or it is possible there is something else configured : https://github.com/jitsi/js-utils/blob/6a4eec6ddad61be6355f542038d9579a50ac6ab0/avatar/index.js#L10

Reply:
I think we currently do use gravatar. I see the only usage of the `getGravatarURL` as:
```
const gravatarBaseURL
                = store.getState()['features/base/config'].gravatarBaseURL ?? 'https://www.gravatar.com/avatar/';

            return getGravatarURL(participant.email, gravatarBaseURL);
```

So we always pass something to the gravatarBaseURL and the default value of https://github.com/jitsi/js-utils/blob/6a4eec6ddad61be6355f542038d9579a50ac6ab0/avatar/index.js#L10 doesn't matter.

I checked our prod environments and we actually have gravatar URLs.

It would be nice to be able to check it in more general way but I don't think there is a decent option. Basically in this mode when downloading resources there are 2 options - CORS or CORP headers. Otherwise the browser won't load them. If we have the CORP header we are fine and we don't need to specify anything. If we have the CORS header (this is the case for gravatar) we need to explicitly add crossOrigin attribute. Potentially we can try to preload them without the attribute, check if it fails, add the attribute and retry. But this would mean additional request and pretty complicated logic because we need to match the attributes later when we add the actual image... 

So I think it is better to add another exception for the things we want to use trough CORS. For now I see only the gravatar failing. If in the future we have a bigger set of URLs maybe there will be a better way to generalise it...